pysages/methods/ffs.py

```diff
@@ -5,7 +5,11 @@
 """
 Forward Flux Sampling (FFS).
 
-Implementation of the direct version of the Forward Flux Sampling algorithm.
+Implementation of the direct version of the FFS algorithm.
```

Comment:
Eventually probably a reference to the FFS paper(s) may be included somewhere.
(PRL 94, 018104 (2005) and/or J. Chem. Phys. 124, 024102 (2006)?).
I guess either here or maybe even better at:
https://github.com/SSAGESLabs/PySAGES/blob/721cc107eb56d617d31166dffcb359cd75b9ca0b/pysages/methods/ffs.py#L76

Reply:
Let's include it now!